### DISCUSSION TITLE
vela core重启后，running状态application也会重新调谐

### DISCUSSION THREAD

**Discussion by EMinEMh** [kubevela/kubevela#5196] [Q&A] [answered]
问题描述：在做vela core性能优化时，改动yaml参数后vela-core pod重启，发现之前running的application都逐渐被重新调谐。因为上线使用有一段时间，所以application资源量还是比较多的，重新调谐花费很多时长，对新application apply体验不友好（无法及时调谐）

我看了下文档（启动参数和性能优化），没有找到能够避免对running的application调谐的参数，请问这种情况有什么办法避免？

vela版本：1.5.4
启动参数：https://kubevela.net/zh/docs/v1.5/platform-engineers/system-operation/bootstrap-parameters
性能优化：https://kubevela.io/zh/docs/platform-engineers/system-operation/performance-finetuning

**Comment by wonderflow** [comment, marked as answer]
@EMinEMh Hi, 升级到 1.5.7 以上版本可以避免此问题 

https://kubevela.net/docs/platform-engineers/system-operation/migration-from-old-version

**Reply by EMinEMh** [reply]
好的收到，非常感谢您的解答
